github/ipld.yml

```diff
@@ -3173,22 +3182,41 @@ teams:
-    #  using a team instead of direct collaborators because we want to reference it in the CODEOWNERS file
+    # Notes: 
+    # 1. These members have push+ access to the github-mgmt repository (in addition to the ipdx team and the org owners listed in "members.admin" above).
+    # 2. This team also has the org-level "moderator" and "security manager" role.
+    #    This is configured through the GitHub UI, not in GitHub management.
+    #    (Org-level role documentation: https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization)
+    # 3. Having a team instead of direct collaborators on the github-mgmt repository also enables easy reference in the github-mgmt CODEOWNERS file.
+    # 4. Leaning on "github-mgmt stewards" for day-to-day admin over true org owners was done
+    #    as part of the effort to reduce org owners in https://github.com/ipfs/ipfs/issues/511
     description: Users that are effectively org admins
     members:
       # WARN: membership here should be treated as cautiously as having an "org owner" role,
       # since one can escalate their privileges accordingly.
       # ATTN: members are expected to:
       #  - be familiar with GitHub Management
       #  - be ready to triage/review org configuration change request in github-mgmt
-      # Intentionally don't have any "maintainers" so that additional membership is done through github-mgmt rather than the GitHub UI.
-      # That said, since most of these people are also "org owners" ("members.admin" above), 
-      # they can still make changes in the UI.
+      # INFO: Intentionally don't have any "maintainers" so that additional membership is done through github-mgmt rather than the GitHub UI.
+      # INFO: There are others who could certainly qualify to be members of this team.  
+      # There is a balance to be had to ensure there are enough knowledgeable people available to support the needs/requests of the github org,
+      # and reducing risk by not having too many with the escalation path that this role affords.
       member:
+        # Why @aschmahmann?
+        # 1. Long-time and still very active contributor to important Go-based IPFS repos that consume and occasionally require changes of IPLD repos, 
+        # 2. Director of IP Shipyard, which is an organization receiving significant grant funding for "IPFS development and maintenance", which includes IPLD as needed.
         - aschmahmann
+        # Why @rvagg?
+        # 1. Continued IPLD maintainer since the pre-2021 days of an "IPLD core team", working across both JS and Go repos.
+        # 2. Active applier of IPLD in contexts like Filecoin.
         - rvagg
+        # Why @vmx?
+        # 1. Continued IPLD maintainer since the pre-2021 days of an "IPLD core team", working across both JS and Rust repos.
+        # 2. Owner of important IPLD community touch points like monthly community calls.
```

Comment:
Sadly those calls are not a thing anymore.

Also I don't really need to be an admin as long as there's some easy way to get things done in case I need something (knowing who to ping is what I consider easy). I'm in favour of having as little admins as it's useful. I don't see it as something "taken away" from me, it just makes sense from an opsec perspective.

Unless you want me as a person that can be pinged and do things, I'm also happy to fill that role for IPLD.